src/components/Section.js

```diff
@@ -17,8 +17,3 @@ export default class Section {
     this._container.prepend(element);
   }
```

Comment:
Needs correction: Use this._items in the methods of Section class, not this._renderedItems